src/test/mochitest/browser.ini

```diff
@@ -138,6 +140,8 @@ skip-if = os == "win"
 [browser_dbg-pretty-print-paused.js]
 [browser_dbg-preview.js]
 skip-if = os == "win"
+[browser_dbg-preview-module.js]
+skip-if = os == "win"
```

Comment:
why do we skip on windows ?

Reply:
windows is weird w/r/t to the mouse events